src/envConfig.ts

```diff
@@ -196,7 +196,6 @@ export const asEnvConfig = asObject({
   KEYS_ONLY_PLUGINS: asOptional(asObject(asBoolean), {}),
   USE_FAKE_CORE: asOptional(asBoolean, false),
   USE_FIREBASE: asOptional(asBoolean, true),
-  USE_WELCOME_SCREENS: asOptional(asBoolean, true),
```

Comment:
Please check with @paullinator about removing this. It might have something to do with whitelabel apps, where we we don't want the Edge-branded welcome scenes to show at all, regardless of our A/B testing.

Reply:
Commit dropped